SubscriptionManagementSystem/persistent-common/src/main/java/com/subsmanagement/entity/SubscriptionPlan.java

```diff
@@ -17,5 +17,5 @@
     private BigDecimal cost;
     private LocalDateTime createTime;
     private LocalDateTime lastUpdateTime;
-    private Integer subscriptionId;
+    private Integer serviceProviderId;
```

Comment:
subscriptions under the same service provider may have different subscription plans to choose?

Reply:
Yes. A service provider should provide multiple plans.